user-interface/src/staff-assignment/screen/StaffAssignmentScreen.test.tsx

```diff
@@ -49,6 +55,13 @@ describe('StaffAssignmentScreen', () => {
     vi.restoreAllMocks();
   });
 
+  test.skip('should call refresh on filter ref when refreshFilter is called', async () => {
```

Comment:
Maybe we should leave a comment or todo about why this is skipped. My recollection is that we were wanting the real ref to call a mock version of the `refreshFilter` function, but that we were unable to do so.